docs/sphinx/developers/wsi.rst

```diff
@@ -2,10 +2,15 @@ Working with whole slide images
 ===============================
 
 Bio-Formats supports many whole slide image formats, but effectively working with this type of data requires some extra considerations.
+Each file contains images representing one or more physical slides, with each slide typically stored at multiple resolutions.
+The width and height of a full resolution whole slide image often exceeds 100,000 pixels, so the uncompressed image size may be several GBs.
+This means that only part of the full resolution image can be accessed at any given time.
 
-The width and height of whole slide images often exceeds 100,000 pixels, so the uncompressed image size may be several GBs.
 JPEG or JPEG-2000 compression is typically used such that the size on disk is often less than 100MB.
-The large image size requires several downsampled versions of the image at progressively smaller resolutions.
+Most whole slide formats split each image into many small tiles of 1024x1024 pixels or smaller and compress each tile independently,
```

Comment:
Done in 020bb49, but I don't see this in the list of conventions in https://github.com/openmicroscopy/ome-documentation, nor does it appear to be the standard across all of our documentation.